chapter-template/chapter-template.md

```diff
@@ -19,24 +18,33 @@ This section should describe the tools or application that must be in place in o
 
 ## Section N
 
-The remainder of the chapter should be broken down in the sections and subsections as needed. These should describe the workflow to be completed as well as the theory or underlying details that provide context to why the workflow is setup as it. 
+The remainder of the chapter should be broken down in the sections and subsections as needed. These should describe the workflow to be completed as well as the theory or underlying details that provide context to why the workflow is setup as it.
 
 - *Are there common command line operations that should be used/understood?*
 - *How do I get started with installing or setting up any required tools?*
 
 ### Example formatting CLI, configuration and code
 
-Begin with a single sentence introducing the snippet and what is accomplishes.
+If needed lead into a code snip with a one line admonition. Keep in mind these do not render in Github style markdown, but will be rendered correctly in the built MkDocs site.
+
+!!! note ""
+    The following shows how to apply using the `systemlinkcli`.
 
 ```bash
 systemlinkcli --apply
 ```
```

Comment:
Do you think it would make sense to include the code snippet inside the admonition as well? That way it groups the description with the snippet nicely. As in:

!!! note ""
    The following shows how to apply using the `systemlinkcli`.

    ```bash
    systemlinkcli --apply
    ```

Reply:
I'm picking up what you're putting down ;-). I've updated how I'm using admonitions to incorporate your suggestion.